### ISSUE TITLE
Import fails

### ISSUE THREAD

**Issue by SCullman** [2sic/2sxc#1234]
**I'm submitting a ...**  
```
- [x ] bug report => search github for a similar issue before submitting 
- [ ] feature request
- [ ] support request => do not submit here, see https://2sxc.org/en/blog/post/moving-discussions-from-codeplex-to-stack-overflow
- [ ] not sure
```

**...about**  
```
- [ ] edit experience / UI
- [ ] admin experience UI
- [ ] Razor templating
- [ ] JS development
- [ ] Content Types or data management
- [ ] APIs like REST
- [ ] DNN parts
- [ x] other / unknown
```

**Current behavior**

- Exported a complete DNN Site
- Exported all apps including content  
"Include all content-groups to re-import the app in an exact copy of this site."
- Deleted 2sxc folder
- Imported all apps again

Importing content app fails with error:    
Could not import the app / package: Die Sequenz enthält kein übereinstimmendes Element.

**Expected behavior**
Import succeeds

**Minimal reproduction of the problem with instructions**
Difficult

**What is the motivation / use case for changing the behavior?**
A better error message/ or the error is catched and ignored

**Please tell us about your environment:**

* **2sxc version(s):**  v09.02.00
* **Browser:** [alli ]
* **DNN:** [ | 8.4.X ] 
* **Language:** [any/all ]

**Anything you would like to add**

The templates/ exports are huge and not optimzed for sharing. Please PM me for details.

**Comment by iJungleboy** [comment]
Sorry for the problem. A few questions
1. Did you export/import from the same dnn/2sxc (same versions)
2. did you delete the folder using explorer or dnn-file manager?
3. Does the export zip look ok to you (is the export failing or the import)?

**Comment by SCullman** [comment]
Hi Daniel,
1. yes, even same instalation, just a different portal
2. Both, doesn't matter
3. Yes, looks fine.

I was able to import everything with newest 9.3.0, but as that testing intallation is missing all my custom modules, I cannot install a new portal with the former portal export.

**Comment by iJungleboy** [comment]
There was a bug in the export - found https://github.com/2sic/2sxc/issues/1285 which exported deleted items - this may be the cause. Could you check? The fix will be in release 09.05.01 today or tomorrow.

**Comment by iJungleboy** [comment]
fyi: published 09.05.01 - pls test, thx! https://github.com/2sic/2sxc/releases/tag/v09.05.01

I'll assume fixed, but if not, I'll re-open this issue.
